src/@next/Select/Select.stories.tsx

```diff
@@ -0,0 +1,265 @@
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+import { withGlintsPortalContainer } from '../../helpers/storybook/Decorators';
+import { InlineError } from '../InlineError';
+
+import { Select, SelectProps } from './Select';
+import { NonSearchableMultiSelect } from './selectStoryHelper/NonSearchableMultiSelect';
+import { SearchableSelect } from './selectStoryHelper/Searchable';
+import { SingleSelect } from './selectStoryHelper/SingleSelect';
+
+(Select as React.FunctionComponent<SelectProps>).displayName = 'Select';
+
+export default {
+  title: '@next/Select',
+  component: Select,
+  subcomponents: {
+    'Select.Label': Select.Label,
+    'Select.OptionList': Select.OptionList,
+    'Select.ActivatorTextInput': Select.ActivatorTextInput,
+    'Select.ActivatorSelect': Select.ActivatorSelect,
+  },
+  decorators: [withGlintsPortalContainer],
+} as Meta;
+
+const countries = [
+  { label: 'Cambodia', value: 'Cambodia' },
+  { label: 'Indonesia', value: 'Indonesia' },
+  { label: 'Malaysia', value: 'Malaysia' },
+  { label: 'Philippines', value: 'Philippines' },
+  { label: 'Singapore', value: 'Singapore' },
+  { label: 'Taiwan', value: 'Taiwan' },
+  { label: 'Thailand', value: 'Thailand' },
+  { label: 'Vietnam', value: 'Vietnam' },
+];
+
+const options = [
+  {
+    active: false,
+    disabled: false,
+    id: '1',
+    label: 'All status',
+    value: 'All status',
+  },
+  {
+    active: false,
+    disabled: false,
+    id: '2',
+    label: 'Completed',
+    value: 'Completed',
+  },
+  {
+    active: false,
+    disabled: false,
+    id: '3',
+    label: 'Pending',
+    value: 'Pending',
+  },
+  {
+    active: false,
+    disabled: false,
+    id: '4',
+    label: 'Expired',
+    value: 'Expired',
+  },
+  {
+    active: false,
+    disabled: false,
+    id: '5',
+    label: 'Cancelled',
+    value: 'Cancelled',
+  },
+];
+
+const NonSearchableSingleSelectTemplate: Story<SelectProps> = args => {
+  return <SingleSelect options={options} {...args} />;
+};
+
+export const NonSearchableSingleSelect = NonSearchableSingleSelectTemplate.bind(
+  {}
+);
+
+NonSearchableSingleSelect.args = {};
+
+const NonSearchableAllowMultiTemplate: Story<SelectProps> = args => {
+  return <NonSearchableMultiSelect options={options} {...args} allowMultiple />;
+};
+
+export const NonSearchableAllowMulti = NonSearchableAllowMultiTemplate.bind({});
```

Comment:
not sure why the color of placeholder is incorrect and doesn't align with other placeholders. it should be #666 instead of #2d2d2d

Reply:
the cause of this one might've been the mix up with the links I shared.